modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/GETTING_STARTED.markdown

```diff
@@ -0,0 +1,186 @@
+# Getting Started with Liferay Workspace
+
+Complete documentation for Liferay Workspace can be found 
+[here](https://learn.liferay.com/dxp/7.x/en/developing-applications/tooling/liferay-workspace.html).
+
+### Folder Structure
+```
+my-project
+├── configs
+│   ├── common
+│   ├── dev
+│   ├── docker
+│   ├── local
+│   ├── prod
+│   └── uat
+├── ext
+│   ├── core ext
+│   └── module exts
+├── modules
+│   ├── apis
+│   ├── services
+│   ├── java widgets
+│   └── js widgets
+├── themes
+│   └── js themes
+└── wars
+    ├── java ee widgets (spring mvc, jsf, etc)
+    └── java themes
+```
+
+### Running Liferay DXP locally
+```
+my-project $ gw initBundle
+my-project $ gw deploy   
+my-project $ blade server run
+```
+### Running Liferay DXP in Docker
+```
+my-project $ gw createDockerContainer 
+my-project $ gw startDockerContainer
+```
+
+### Creating a Liferay DXP distribution
+
+#### Creating a tar
+```
+my-project $ gw distBundleTar
+```
+
+#### Creating a zip
+```
+my-project $ gw distBundleZip
+```
+
+#### Creating a docker image
+```
+my-project $ gw buildDockerImage
+```
+
+### Create a Liferay DXP module
+
+#### Creating a new service
+```
+my-project $ blade create -t service-builder my-service
+```
+
+#### Creating a javascript theme
+```
+my-project $ blade create -t js-theme my-theme
+```
+
+#### Creating a Java Widget
+```
+my-project $ blade create -t mvc-portlet "my-java-widget"
+```
+
+#### Create a JS Widget
+```
+my-project $ blade create -t js-widget "my-js-widget"
+```
+
+### Gradle Properties
+
+Set the following in `gradle.properties` to change the default settings.
+
+#### liferay.workspace.product
+Set the `liferay.workspace.product` to set the `app.server.tomcat.version`, 
+`liferay.workspace.bundle.url`, `liferay.workspace.docker.image.liferay`, and 
+`liferay.workspace.target.platform.version` that matches your Liferay Product 
+Version. To override each of these settings, set them individually. 
+
+#### app.server.tomcat.version
```

Comment:
should we sort these properties in to "most likely to set/need" sorted to "least likely"?  Like if we take the stance that eventually they may stop reading this markdown doc, we woul want the most important properties "above the fold".

Reply:
But maybe you already did that, this order isn't that bad, just thought that app.service.tomcat.version would be lower, but i think the properties that we have in "product_info" are already the "most important" to customers. so that would be how we have it.  But for instances "bundles.cache.dir" should be lower in this doc than something like `liferay.workspace.environment`

Reply:
I switched the ordering to be product_info related keys alphabetically... then the rest alphabetically.  We can add some headings to make it clearer... but we need some easy sorting other than priority because it will be very difficult for others to figure it out.